packages/stacking/README.md

```diff
@@ -145,3 +158,99 @@ stackingStatus:
 //   }
 // }
 ```
+
+## Delegation
+
+There are four methods available for delegation, two for the delegators and two for the delegatee.
+
+### Delegatee
+
+If you are the account owner / stacker / delegatee, you can delegate or revoke delegation rights.
+
+#### Delegate STX
+
+```typescript
+// STX address of the delegator 
+const delegateTo = "ST2MCYPWTFMD2MGR5YY695EJG0G1R4J2BTJPRGM7H";
+// burn height at which the delegation relationship should be revoked (optional)
+const untilBurnBlockHeight = 5000;
+
+const delegetateResponse = await client.delegateStx({
+  amountMicroStx,
+  delegateTo,
+  untilBurnBlockHeight, // optional
+  poxAddress, // optional
+  privateKey,
+});
+
+// {
+//   txid: '0xf6e9dbf6a26c1b73a14738606cb2232375d1b440246e6bbc14a45b3a66618481',
+// }
+```
+
+#### Revoke delegation
+
+```typescript
+// note that the parameter here is not JSON
+const revokeResponse = await client.revokeDelegateStx(privateKey);
+
+// {
+//   txid: '0xf6e9dbf6a26c1b73a14738606cb2232375d1b440246e6bbc14a45b3a66618481',
+// }
+```
+
+### Delegator
+
+If you are the delegator, you can stack / lock up tokens for your delegatees and commit to stacking participation for upcoming reward cycles.
+
+#### Stack delegated STX
+
+```typescript
+// delegators would initiate a different client
+const delegatorAddress = "ST22X605P0QX2BJC3NXEENXDPFCNJPHE02DTX5V74";
+// delegator private key for transaction signing
+const delegatorPrivateKey = 'd48f215481c16cbe6426f8e557df9b78895661971d71735126545abddcd5377001';
+// a BTC address for reward payouts to the delegator
+const deletagorPoxAddress = "msiYwJCvXEzjgq6hDwD9ueBka6MTfN962Z";
+
+const delegatorClient = new StackingClient(delegatorAddress, network);
+
+// STX addresses of all your delegatees / stackers
+const stackers = [address];
+
+
+// note this does not require an await
```

Comment:
That can't be right - how is this not async? Is the transaction not broadcasted?

From reading the implementation, this is async

Reply:
oh, you're right!